stress/src/logs.rs

```diff
@@ -37,14 +39,15 @@ pub struct MockLogProcessor {
 impl LogProcessor for MockLogProcessor {
     fn emit(&self, record: &mut opentelemetry_sdk::logs::LogRecord, scope: &InstrumentationScope) {
         let log_tuple = &[(record as &LogRecord, scope)];
-        let _ = futures_executor::block_on(self.exporter.export(LogBatch::new(log_tuple)));
+        let log_batch = LogBatch::new(log_tuple);
```

Comment:
nit: we can keep existing code to reduce the amount of diffs in PR.

Reply:
Sure. done.